src/Drupal/Driver/Fields/Drupal8/ImageHandler.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\Driver\Fields\Drupal8;
+
+/**
+ * Image field handler for Drupal 7.
+ */
+class ImageHandler extends AbstractHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function expand($values) {
+    $data = file_get_contents($values[0]);
+    if (FALSE === $data) {
+      throw new \Exception("Error reading file");
+    }
+
+    /* @var \Drupal\file\FileInterface $file */
+    $file = file_save_data(
+      $data,
+      'public://' . uniqid() . '.jpg');
+
+    if (FALSE === $file) {
+      throw new \Exception("Error saving file");
+    }
+
+    $file->save();
+
+    $return = array(
+      'target_id' => $file->id(),
+      'alt' => 'Behat test image',
```

Comment:
Is there a way to make these the default value (`alt` and `title`) in case somebody wanted to test with actual values?